e2e-test/src/test/java/org/alfresco/hxi_connector/e2e_test/AgentsE2eTest.java

```diff
@@ -91,6 +100,22 @@ void shouldReturnAgents()
         assertEquals(expected, response.jsonPath().get("list.entries"));
     }
 
+    @Test
+    void shouldReturnAgentAvatar() throws IOException
```

Comment:
Shouldn't we also cover more scenarios via E2E tests? 

E.g.:
- trying to request an avatar other than `-default-` _(that's already unit-tested, but I'd vote in favor of having the E2E test verify it as well so it can also verify the **response status code**!)_
- trying to request an avatar for an agentId that doesn't exist

Reply:
Added 👍